pingouin/parametric.py

```diff
@@ -611,7 +612,11 @@ def rm_anova(data=None, dv=None, within=None, subject=None, correction='auto',
 
     aov = aov.reindex(columns=col_order)
     aov.dropna(how='all', axis=1, inplace=True)
-    return _postprocess_dataframe(aov)
+    aov = _postprocess_dataframe(aov)
+
+    aov.residuals_ = 0
```

Comment:
Alternatively, we could use [pandas.DataFrame.attrs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.attrs.html), e.g:

```python
df.attrs = dict(residuals=resid)
```

although I have never tried it so I don't know if it works well